src/mono/wasm/runtime/jiterpreter-support.ts

```diff
@@ -505,6 +528,18 @@ export class WasmBuilder {
             this.appendULeb(0x01);
         }
 
+        if (enableWasmEh) {
+            // import the c++ exception tag
+            this.appendName("x");
+            this.appendName("e");
+            // tagtype
+            this.appendU8(0x04);
```

Comment:
please make those named constants

Reply:
these are binary constants from the webassembly spec's binary format, the comment above them explains what they are.